### PR TITLE
Add stitches + Refactor components

### PR DESCRIPTION
**Pull request by sean-binary**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Addresses (redmine-issue-number)#58167, #58274, #58275

## Type of change

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update
-   [ ] Package update

# Author Checklist:

Please check relevant items before requesting for review.

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] My changes generate no new warnings
-   [x] I have added tests that prove my fix is effective or that my feature works
-   [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by maryia-deriv — DISMISSED** [review]

File: webpack.config.ts [hunk 1]
```diff
@@ -62,6 +62,9 @@ const config = (env: EnvConfig): Configuration => {
         },
         resolve: {
             extensions: ['.tsx', '.ts', '.js'],
+            alias: {
+                'Styles': path.resolve(__dirname, 'src/styles'),
```
Comment (balakrishna-binary) [inline webpack.config.ts:66]: I thought we are going to maintain the alias in one place - tsconfig.json after we added `TsconfigPathsPlugin`

Comment (sean-binary) [reply webpack.config.ts:66]: Ah yes, this was before I pulled the latest code from @farrah-deriv 👍 Let me remove that.

Comment (sean-binary) [reply webpack.config.ts:66]: Will copy cover the config from her PR 🙏

**Review by balakrishna-binary — APPROVED** [review]

**Comment by balakrishna-binary** [comment]
LGTM. Nice work 👏

**Pull request merged**